serializer.rst

```diff
@@ -19,7 +19,7 @@ install the serializer before using it:
 
 .. code-block:: terminal
 
-    $ composer require symfony/serializer
+    $ composer require serializer
```

Comment:
We've currently decided (though it's not uniformly implemented) to not rely on the Flex aliases in the docs. So, this would be `composer require symfony/serializer-pack`

Reply:
I'll update it.